### ISSUE TITLE
IIA-ID not required in Omobility GET response ?

### ISSUE THREAD

**Issue by skishk** [erasmus-without-paper/ewp-specs-api-mobilities#55]
I saw that is possible to receive a nomination from partner without IIA-ID in the Omobility GET-RESPONSE:

https://github.com/erasmus-without-paper/ewp-specs-api-omobilities/blob/9e48843951c760af4bf1958e0f94dda7b381b588/endpoints/get-response.xsd#L206

am I wrong? or am I missing something? or I understand well ? someone could help me to understand that? 

thanks in advance

**Comment by mkurzydlowski** [comment]
You are right that this element is optional. What is also currently optional is the `iia-id` element in the `sending-hei` tree.

**Comment by janinamincer-daszkiewicz** [comment]
In the very early period of the EWP project we also considered possibility to make this API available to so called free movers. In particular it was assumed that some HEIs may implement this API and do not implement IIAs or LAs. This should not happen if we take into account processes requred by the Erasmus+ programme. Which means that the status of the two mentioned attributes may change during the next round of upgrades related to nominations.

**Comment by skishk** [comment]
thank everyone for answering .

I think yes is better these fields are required because if someone send nomination with no iia-id it could be rejected automatically in my opinion.

**Comment by skishk** [comment]
so the examples too need a revision if IIA-ID become mandatory.

if someone develop this API using only the examples (like it happened with IIA) he will miss a lot of different scenarios

**Comment by janinamincer-daszkiewicz** [comment]
We will definitely come back to nominations and ToR after we finish with IIAs and LAs.
